game-core/src/main/java/games/strategy/engine/lobby/server/db/HashedPassword.java

```diff
@@ -25,11 +25,11 @@ public boolean isHashedWithSalt() {
   }
 
   public boolean isBcrypted() {
-    return value != null && value.matches("^\\$2a\\$.{56}$");
+    return value.matches("^\\$2a\\$.{56}$");
```

Comment:
The `null` guard should not be necessary because the constructor always converts `null` to an empty string.